test/integration/deploy_scale_test.go

```diff
@@ -104,7 +106,11 @@ func TestDeployScale(t *testing.T) {
 		if err != nil {
 			return false, nil
 		}
-		return appsinternalutil.HasSynced(config, generation), nil
+		externalConfig := &appsv1.DeploymentConfig{}
```

Comment:
why don't we use v1 client?

Reply:
we can

Reply:
@tnozicka mind a follow up? this will require reworking half of that test